sycl/test-e2e/ESIMD/api/functional/ctors/ctor_fill_core.cpp

```diff
@@ -5,7 +5,10 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-// RUN: %{build} -fsycl-device-code-split=per_kernel -o %t.out
+// TODO: remove fno-fast-math option once a compiler issue resulting
+// incorrect execution results when using fast-math is resolved.
```

Comment:
I do not understand what compiler issue is referred to here.
If the GPU does not obey `fast-math` flag, but host does, there is no compiler issue.

Reply:
Is this a GPU hardware limitation, or something that is missing in the GPU compiler?